<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>740</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Flightgear AI schedule manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/icons/main.png</normaloff>:/icons/icons/main.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>1061</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="flightsTab">
     <attribute name="title">
      <string>Flights</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>180</y>
        <width>1041</width>
        <height>431</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>840</width>
        <height>291</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1200</width>
        <height>436</height>
       </size>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <column>
       <property name="text">
        <string>Callsign</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Flight rules</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dep. airport</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Arr. airport</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dep. time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Arr. time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dep. day</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Aircraft type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Flt. level</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>1041</width>
        <height>176</height>
       </rect>
      </property>
      <property name="title">
       <string>Filters</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>991</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="callsignEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Callsign</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Dep. airport</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="depAirportEdit"/>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Dep. time</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="depTimeEdit"/>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Dep. day</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLineEdit" name="depDayEdit"/>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Ac. type</string>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QLineEdit" name="acTypeEdit"/>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="arrTimeEdit"/>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Arr. time</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="arrAirportEdit"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Arr. airport</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Airline</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="airlineEdit"/>
        </item>
        <item row="1" column="7">
         <widget class="QPushButton" name="showButton">
          <property name="text">
           <string>Show</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/icons/dialog-ok-apply.png</normaloff>:/icons/icons/dialog-ok-apply.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/icons/user-busy.png</normaloff>:/icons/icons/user-busy.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>525</x>
         <y>126</y>
         <width>476</width>
         <height>46</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="editButton">
          <property name="text">
           <string>Edit flight</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/icons/document-edit.png</normaloff>:/icons/icons/document-edit.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>Delete flights</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/icons/application-exit.png</normaloff>:/icons/icons/application-exit.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="truncateButton">
          <property name="text">
           <string>Empty database</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/icons/system-reboot.png</normaloff>:/icons/icons/system-reboot.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>125</y>
         <width>116</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Total nr. of flights:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>106</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Selected flights:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelTotalFlights">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>125</y>
         <width>56</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelSelectedFlights">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>66</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="aircraftTab">
     <attribute name="title">
      <string>Aircraft</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>1051</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>&amp;Import...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>A&amp;bout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>callsignEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>showButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>arrAirportEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>showButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acTypeEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>showButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>956</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>airlineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>showButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>arrTimeEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>showButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>depAirportEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>showButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>depDayEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>showButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>764</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>depTimeEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>showButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
